atd-etl/prefect/afd_ems_incidents/afd_incident_upload.py

```diff
@@ -0,0 +1,279 @@
+#!/usr/bin/env python
+
+"""
+Name: AFD Incident Uploads
+Description: This flow uploads AFD Incident Response CSVs (AFD Contact: Gus). 
+    The data is emailed to atd-afd-incident-data@austinmobility.io daily ~ 6:30AM. From there it
+    gets forwarded to a S3 bucket via AWS Simple Email Service.
+"""
+
+import prefect
+from prefect import Flow, task, Parameter, case
+import datetime
+import json
+import boto3
+import pprint
+import os
+import time
+import email
+import tempfile
+import shutil
+import pandas
+from datetime import datetime, timedelta
+import psycopg2
+import psycopg2.extras
+from prefect.backend import get_key_value
+
+pp = pprint.PrettyPrinter(indent=4)
+
+kv_data = get_key_value(key="Vision Zero Development")
```

Comment:
Suggestion to put this in a task that way you won't have to re-register your flow if any of your key values get updated. Could also accept a parameter to change it from dev to production?

Something like this?
```
@task
def get_env_vars(env):
    if env == "prod":
       environment_variables = get_key_value(key="Vision Zero")
   else:
      environment_variables = get_key_value(key="Vision Zero Development")
    return environment_variables

```

Reply:
I'm not sure if we should make it switch between production or not, but I 100% agree that the KV retrieval should be pulled into a task. I'll update the PR with that - 🙏